charts/sda-svc/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: v2
 name: sda-svc
-version: "0.20.7"
+version: "0.21.0"
+appVersion: "v0.0.163"
 kubeVersion: ">= 1.19.0-0"
```

Comment:
should this be bumped in all charts ? to `1.25`

Reply:
Soon, we need to remove everything that has to do with PSP first, but that will be a different PR since it will have implications for the current deployments.